src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -70,9 +70,10 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     public List<String> primaryAfflFieldNames;
 
     /*******************************************************************************************************
-    * @description Map of Account Record Type Label to API
+    * @description Static Map of Account Record Type Label to API
```

Comment:
I am not convinced why we are caching this variable. Are we caching it only to provide access to Exception handler class? If we are passing the map as parameter, why do we still need to cache it? IMHO we should avoid using static variables if necessary since it is global state and can be difficult to debug.

Reply:
We are caching this because it is schema information populated by a query. The appropriate place to do this caching post-refactor would be in whatever replaces UTIL_Describe. For the time being, this 'quick' fix is meant to ameliorate needing to perform a SOQL query to get this information more than once a transaction.